app/assets/javascripts/admin/addon/templates/web-hooks-edit.hbs

```diff
@@ -0,0 +1,105 @@
+<LinkTo @route="adminWebHooks" class="go-back">
```

Comment:
A page title here would be good too. Webhooks don't have names, maybe we can truncate the webhook's URL? Not sure...

Reply:
Skipping this one for now, doesn't seem needed that much there. Some other edit pages don't have title elements either. But we can revisit that, I already have a second batch of improvements in this area, so I can add it there if needed.